lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -287,15 +287,49 @@ def get_stack_events(cfn, stack_name):
         return ret
 
     for e in events.get('StackEvents', []):
-        eventline = 'StackEvent {ResourceType} {LogicalResourceId} {ResourceStatus}'.format(**e)
-        ret['events'].append(eventline)
-
-        if e['ResourceStatus'].endswith('FAILED'):
-            failline = '{ResourceType} {LogicalResourceId} {ResourceStatus}: {ResourceStatusReason}'.format(**e)
-            ret['log'].append(failline)
+        if e['Timestamp'] > started_at:
```

Comment:
Interesting idea!

Unfortunately single run can contain multiple events ending with `_COMPLETE`. Here an example from single run:

```
10:05:02 UPDATE_ROLLBACK_COMPLETE	AWS::CloudFormation::Stack
10:05:01 UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS
...
10:04:01 UPDATE_ROLLBACK_IN_PROGRESS	AWS::CloudFormation::Stack
10:03:01 UPDATE_FAILED	    AWS::Route53::RecordSet DNSRecord2
10:03:01 UPDATE_COMPLETE	AWS::Route53::RecordSet DNSRecord1
...
10:01:01 UPDATE_IN_PROGRESS	AWS::Route53::RecordSet	DNSRecord2 (for subresources)
10:01:01 UPDATE_IN_PROGRESS	AWS::Route53::RecordSet	DNSRecord1 (for subresources)
10:01:00 UPDATE_IN_PROGRESS AWS::CloudFormation::Stack (the first one we want to see
09:01:00 UPDATE_ROLLBACK_COMPLETE (from previous run, not interesting for us)
```

BTW, timezone ~solution~ hack ;-) is almost ready, should work with Python 2.6, 2.7, 3.x
Will commit soon